src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -718,6 +718,7 @@ public Completable invoke(String method, Object... args) {
 
     /**
      * Invokes a hub method on the server using the specified method name and arguments.
+     * A Type can be retrieved from a Class by using {@link TypeReference}
```

Comment:
Nit: Is it really "from a Class"?

```suggestion
     * A Type can be retrieved by using {@link TypeReference}
```